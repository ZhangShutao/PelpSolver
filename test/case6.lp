q(a) | -q(a).
p(X) :- q(X), K[0, 1) not q(X).