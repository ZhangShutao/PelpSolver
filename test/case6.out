WorldView 1:
{[0.5]:{-q(a)},[0.5]:{p(a),q(a)}}